magicctapipe/io/io.py

```diff
@@ -92,9 +133,9 @@ def format_object(input_object):
     string = string.replace("'", "").replace(",", "")
 
     return string
+    
 
-
-def get_stereo_events(
+def get_stereo_events_old(
```

Comment:
you can get rid of the need of two functions if you allow for two parameters: config file or tel_combinations, and check which one is not null

Reply:
The second function will be deleted after the second PR (RFs and DL2), so I don't know if it is worth to change the new function to include the second one and then change it again in the next PR. I will discuss about this with Raniere and Federico

Reply:
ok, I'm closing this for now. we might come back to this with the other PR